bundles/org.openhab.binding.airq/src/main/resources/OH-INF/thing/thing-types.xml

```diff
@@ -0,0 +1,660 @@
+		<item-type>Number</item-type>
+		<label>Absolute Humidity</label>
+		<state readOnly="true" pattern="%.3f g/m³"></state>
+	</channel-type>
+
+	<channel-type id="mtime" advanced="true">
+		<item-type>Number</item-type>
+		<label>Time Needed for Measurement</label>
+		<state readOnly="true" pattern="%f ms"></state>
+	</channel-type>
+
+	<channel-type id="no2" advanced="false">
+		<item-type>Number</item-type>
+		<label>NO₂ Concentration</label>
+		<state readOnly="true" pattern="%.2f μg/m³"></state>
+	</channel-type>
+
+	<channel-type id="o3" advanced="false">
+		<item-type>Number</item-type>
+		<label>O₃ Concentration</label>
+		<state readOnly="true" pattern="%.2f μg/m³"></state>
+	</channel-type>
+
+	<channel-type id="oxygen" advanced="false">
+		<item-type>Number:Dimensionless</item-type>
+		<label>Oxygen Concentration</label>
+		<state readOnly="true" pattern="%.3f %%"></state>
+	</channel-type>
+
+	<channel-type id="performance" advanced="false">
+		<item-type>Number:Dimensionless</item-type>
+		<label>Performance Index</label>
+		<state readOnly="true"></state>
+	</channel-type>
+
+	<channel-type id="pm1" advanced="false">
+		<item-type>Number</item-type>
+		<label>Fine Dust Concentr. >1μ</label>
+		<state readOnly="true" pattern="%.0f μg/m³"></state>
+	</channel-type>
+
+	<channel-type id="pm10" advanced="false">
+		<item-type>Number</item-type>
+		<label>Fine Dust Concentr. >10μ</label>
+		<state readOnly="true" pattern="%.0f μg/m³"></state>
+	</channel-type>
+
+	<channel-type id="pm2_5" advanced="false">
+		<item-type>Number</item-type>
+		<label>Fine Dust Concentr. >2,5μ</label>
+		<state readOnly="true" pattern="%.0f μg/m³"></state>
+	</channel-type>
+
+	<channel-type id="pressure" advanced="false">
+		<item-type>Number:Pressure</item-type>
+		<label>Pressure</label>
+		<state readOnly="true" pattern="%.2f hPa"></state>
+	</channel-type>
+
+	<channel-type id="so2" advanced="false">
+		<item-type>Number</item-type>
+		<label>SO₂ Concentration</label>
+		<state readOnly="true" pattern="%.2f μg/m³"></state>
+	</channel-type>
+
+	<channel-type id="sound" advanced="false">
+		<item-type>Number</item-type>
+		<label>Noise</label>
+		<state readOnly="true" pattern="%.1f db(A)"></state>
+	</channel-type>
+
+	<channel-type id="temperature" advanced="false">
+		<item-type>Number:Temperature</item-type>
+		<label>Temperature</label>
+		<state readOnly="true" pattern="%.2f %unit%"></state>
+	</channel-type>
+
+	<channel-type id="timestamp" advanced="false">
+		<item-type>DateTime</item-type>
+		<label>Time Stamp</label>
+		<state readOnly="true"></state>
+	</channel-type>
+
+	<channel-type id="tvoc" advanced="false">
+		<item-type>Number</item-type>
+		<label>VOC Concentration</label>
+		<state readOnly="true" pattern="%.0f ppb"></state>
+	</channel-type>
+
+	<channel-type id="uptime" advanced="true">
+		<item-type>Number</item-type>
+		<label>Uptime</label>
+		<state readOnly="true" pattern="%.0f s"></state>
```

Comment:
```suggestion
		<state readOnly="true" pattern="%.0f %unit%"></state>
```

Reply:
This is not a time but a period, given in seconds. It seems that openHAB can handle time, but not periods (eg showing 120s as 2m). Or do you have another idea on how to handle this?
If openHAB can not handle time periods, I would prefer to have the unit displayed as seconds, otherwise the user does not know the unit.

Reply:
`Number:Time` is a period. `DateTime` would be a point in time. If you prefer a special unit, you can leave the `s`.

Reply:
Thank you. I put `Number:Time`, but left `s` as a unit because otherwise openHAB just shows the number, without any unit.

Reply:
I would consider this a bug if no unit is displayed, although you specified `%unit%`. You could open an issue here https://github.com/openhab/openhab-core/issues

Reply:
This isn't a bug in the framework. You need to send `QuantityType` instead of `DecimalType` if you want to use Units of Measure. See https://www.openhab.org/docs/concepts/units-of-measurement.html

Reply:
OK. I tried the following:
xml file:
```
	<channel-type id="uptime" advanced="true">
		<item-type>Number:Time</item-type>
		<label>Uptime</label>
		<state readOnly="true" pattern="%d %unit%"></state>
	</channel-type>
```
code:
```
        updateState(channelName,
                new QuantityType<Time>(Integer.parseInt(dec.get(airqName).toString()), Units.SECOND));
```
However, it still does not work, the output it still just a number, without a unit. I have no experience in Parameterized Types (<>) so it would be the last thing assuming a bug - rather it is a deficiency from my side. I tried several variants to solve this, but I need your help. Where is the bug in above code? I would really love using this feature because abstracting a unit is a very good idea, but I'm stuck here.

Reply:
Solved this, it is shown properly. The code above was correct, but I had not deleted the Item and created a new one. By recreating the item it works now.